apps/prairielearn/src/schemas/infoAssessment.ts

```diff
@@ -43,31 +43,54 @@ export const AssessmentAccessRuleJsonSchema = z
       .describe(
         'The PrairieTest exam UUID for which a student must be registered. Implies mode: Exam.',
       )
-      .optional(),
+      .optional()
```

Comment:
Every default added corresponds to the removal of a `?? <val>` from `sync/fromDisk/x.ts`.